cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -145,6 +145,7 @@ pub type SignedExtra = (
 		bridge_to_bulletin_config::OnBridgeHubRococoRefundRococoBulletinMessages,
 	),
 	cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim<Runtime>,
+	frame_metadata_hash_extension::CheckMetadataHash<Runtime>,
```

Comment:
@yrong this breaks the bridge zombienet tests. I can either remove it here or add it to the bridge relayer in order to address this issue. So just wanted to ask if you actually need it here in order to understand how to fix this. The same for bridge hub westend.

Also I think the runtime version should be incremented 

Related to: https://github.com/paritytech/polkadot-sdk/issues/5296